rubocop-rspec.gemspec

```diff
@@ -37,6 +37,6 @@ Gem::Specification.new do |spec|
     'rubygems_mfa_required' => 'true'
   }
 
-  spec.add_runtime_dependency 'rubocop', '~> 1.33'
+  spec.add_runtime_dependency 'rubocop', '>= 1.33', '< 2.0'
```

Comment:
This means exactly the same thing:

- https://guides.rubygems.org/patterns/#declaring-dependencies
- https://thoughtbot.com/blog/rubys-pessimistic-operator

So there is no difference in behavior.